lib/mintel/rules/mintel-workload.libsonnet

```diff
@@ -0,0 +1,37 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'mintel-workload.rules',
+        rules: [
+          {
+            expr: |||
+              sum(
+                  node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate
```

Comment:
If we use a pre-recording we need to make sure this is evaluated AFTER the other one